sources/packages/backend/workflow/test/2022-2023/parttime-assessment/appeals/parttime-assessment-student-financial-information.e2e-spec.ts

```diff
@@ -0,0 +1,55 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import {
+  createFakeStudentDependentEligibleForChildcareCost,
+  DependentChildCareEligibility,
+} from "../../../test-utils/factories";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-student-financial-information.`, () => {
+  it(
+    "Should have calculated student financial information variables assigned with request a change values " +
+      "when there is a request a change for student financial information.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataDependants = [
+        createFakeStudentDependentEligibleForChildcareCost(
+          DependentChildCareEligibility.Eligible0To11YearsOld,
+          assessmentConsolidatedData.offeringStudyStartDate,
+        ),
+        createFakeStudentDependentEligibleForChildcareCost(
+          DependentChildCareEligibility.Eligible12YearsAndOver,
+          assessmentConsolidatedData.offeringStudyStartDate,
+        ),
+      ];
+      assessmentConsolidatedData.appealsStudentFinancialInformationAppealData =
```

Comment:
`appealsStudentFinancialInformationAppealData` can be typed in the same way that we also have `StudentDependent` type defined in the `assessmentConsolidatedData`.